webpack/assets/javascripts/react_app/components/EditableTextDisplay/EditableTextDisplay.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Spinner,
+  TextInput,
+  TextArea,
+  Text,
+  TextVariants,
+  Button,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+import { TimesIcon, CheckIcon, PencilAltIcon } from '@patternfly/react-icons';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+import './editableTextDisplay.scss';
+
+const EditableTextDisplay = ({ onEdit, value, textArea, attribute }) => {
```

Comment:
Why don't we call the component `InlineEdit`? :thinking:
I was just confused when reading the component name for the first time, so I'm thinking about a better name here.
```suggestion
const InlineEdit = ({ onEdit, value, textArea, attribute }) => {
```